drift:
- -0.005303653239904371
- 0.0018212786518062493
- 0.009486121476160365
pose_position_z_covariance: 0.00001